### DISCUSSION TITLE
Update AMP Optimizer to version 2.8.x

### DISCUSSION THREAD

**Discussion by cesarmiquel** [vercel/next.js#29422] [Ideas]
### Describe the feature you'd like to request

Looking at the source code it looks like NextJS is using version **2.7.1-alpha.0** of the AMP Optimizer. That version dates from December 2020 and is an alpha version. In the meantime there has been several 2.7.x official releases and since May 2021 there is a 2.8.x official release with several enhancements and bug fixes.  (see https://github.com/ampproject/amp-toolbox/releases/tag/v2.8.0)

### Describe the solution you'd like

It would be nice to update NextJS to the latest stable version of the AMP optimizer to take advantage of these fixes and improvements. Ideally I would upgrade to the 2.8.3 which is the latest version at this point.

### Describe alternatives you've considered

If we don't want to upgrade to the 2.8.x branch at least update to the latest 2.7.x but I would much prefer we have a new version to take advantage of said improvements.

**Comment by timneutkens** [comment]
Feel free to open a PR upgrading it 👍

**Reply by cesarmiquel** [reply]
Ok. I'll create a PR. Besides bumping the version and running all tests is there something else I should look at?
